### DISCUSSION TITLE
preface

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#118] [Announcements]
# preface

An unofficial and opinionated book for beginners

https://nixos-and-flakes.thiscute.world/preface

**Comment by soymadip** [comment]
wow looks like finally found something to learn nix

**Comment by gniw** [comment]
All this time, I’ve been longing for this. This is the best!

**Comment by TheArctesian** [comment]
Thank you so much for book, really well put together and has helped me immensely.

**Comment by chroy2** [comment]
I love this book, it has been a huge help with Nix. Thank you for creating this great tool.
 
<(^_^)> chroy2

**Comment by Shadur** [comment]
Oh boy, time to dig in.

**Comment by ZenCiphre** [comment]
such a peaknuke!
